web_hosting_manager/app/components/app_logs.js

```diff
@@ -14,27 +14,27 @@ export default class AppLogs extends Component {
     };
   }
   componentDidMount() {
-    this.setState({loading: true});
-    this.props.getLogFilePath()
-      .then(path => {
+    this.setState({ loading: true }); // eslint-disable-line react/no-did-mount-set-state
```

Comment:
instead of disabling linting here, can we rework this so that it's meeting the linter's requirements.

Do we need to be setting state in mount? Could it happen in the constructor?

Should we maybe be triggering global `redux` state changes instead of local to the component? This is afterall some part of the whole app state, right?

Reply:
Thank you josh. I'm updating the correction and will push it.